sql/derived_tables/instance_ext.sql

```diff
@@ -7,18 +7,23 @@ SELECT
     instance.id AS instance_id,
     instance.hrid AS instance_hrid,
     json_extract_path_text(instance.data, 'catalogedDate') AS cataloged_date,
-    instance.discovery_suppress,
+    instance.title,
     instance.index_title,
     instance.instance_type_id AS type_id,
     instance_type.name AS type_name,
     instance.mode_of_issuance_id,
     mode_of_issuance.name AS mode_of_issuance_name,
     json_extract_path_text(instance.data, 'previouslyHeld')::boolean AS previously_held,
     instance.source AS instance_source,
-    instance.staff_suppress,
+    json_extract_path_text(INSTANCE.DATA, 'discovery_suppress')::boolean AS discovery_suppress,
+    json_extract_path_text(INSTANCE.DATA, 'staff_suppress')::boolean AS staff_suppress,
```

Comment:
I' think the json names should be `staffSuppress` and `discoverySuppress`

Reply:
Thanks Axel for catching that. I just made the change. I also saw that it was still uppercase which I also just fixed.